src/permissions/PermissionCallableAllowedContractNativeTokenRecurringAllowance.sol

```diff
@@ -104,7 +105,7 @@ contract PermissionCallableAllowedContractNativeTokenRecurringAllowance is
                 if (withdraw.asset != address(0)) revert InvalidWithdrawAsset(withdraw.asset);
                 // do not need to accrue callsSpend because withdrawn value will be spent in other calls
             } else if (call.data.length == 0) {
-                // only allow direct ETH transfer for debiting paymaster (optional)
+                // only allow direct ETH transfer for crediting paymaster (optional)
```

Comment:
the user certainly isn't "debiting" the paymaster/taking funds from, right?

Reply:
I guess maybe it's not debiting the paymaster in that sense, but also the user is not crediting the paymaster. The paymaster is crediting the user because it's paying on the user's behalf and hoping to get paid back for extending that line of credit.

Maybe I should just find a different word if it's not obvious. How about "refunding" paymaster?